### DISCUSSION TITLE
unable to play music

### DISCUSSION THREAD

**Discussion by kushagrasingh772** [Rapptz/discord.py#6014] [Q&A]
whenever is use the command .yt "Song_name" or .stream "Song_name" i get an error "Failed to parse JSON  (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)'))". I can play music by typing the url, but its not working when i just write the title

**Comment by Harmon758** [comment]
There isn't sufficient information here to provide much help for you.
All that can be determined is, as the error says, there's some JSON that's failing to be parsed.
You'll need to provide the actual code and/or a full traceback for more help.
